io.github.garglk.Gargoyle.yaml

```diff
@@ -0,0 +1,31 @@
+id: io.github.garglk.Gargoyle
+runtime: org.kde.Platform
+runtime_version: "5.15-21.08"
+sdk: org.kde.Sdk
+command: gargoyle
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --socket=system-bus
```

Comment:
no. you should figure out which d-bus name it needs. use `--log-system-bus` with `flatpak run` to figure out what is needed.

Reply:
As mentioned, this was to get feedback on an error message.  There's nothing that's output with `--log-system-bus`, even though the error message only disappears when there's a system bus (session bus isn't enough, though there _is_ session bus logging).  As in, providing access to the system bus is the only way to remove the error, but there's apparently nothing accessing the system bus, according to the logging?  Not sure what's happening here, and it's possible it's some idiosyncratic behavior on my system.

Again, this doesn't _seem_ to affect behavior, but I'd obviously like to determine why the error message is there.

Reply:
Could you try `--system-talk-name=org.freedesktop.UDisks2`?

Reply:
That seems to do the trick, thanks!